README.md

```diff
@@ -25,6 +25,23 @@ See the [Electron docs for using native modules][electron-native-modules] to ens
 [electron-rebuild]: https://github.com/electron/electron-rebuild
 [electron-native-modules]: https://www.electronjs.org/docs/tutorial/using-native-node-modules
 
+### Known problems using with electron:
+
+- If you try to use `usb-detection` with [electron](https://www.electronjs.org/), you run into the following error. You can follow the steps below to resolve the problem.
```

Comment:
```suggestion
If you try to use `usb-detection` in Electron without recompiling using `electron-rebuild`, you will encounter an error similar to:
```

Reply:
@MadLittleMods and @mcous this way look accurate for you?

### Additional information:
- If you try to use `usb-detection` in Electron without recompiling using `electron-rebuild`, you will encounter an error similar to:
    ```
    detection.node was compiled against a different Node.js version using NODE_MODULE_VERSION 72. This version of Node.js requires NODE_MODULE_VERSION 80. Please try re-compiling or re-installing
    ```

    Is possible that your project isn't correctly set up. You can try to solve this problem following next steps:

    1. `npm i electron-rebuild --save-dev`
    1. `./node_modules/.bin/electron-rebuild`

Reply:
@carboleda The original wording sounds more clear to me, https://github.com/MadLittleMods/node-usb-detection/pull/118/files#r493182159